insights/tests/test_soscleaner.py

```diff
@@ -35,6 +35,42 @@ def test_sub_ip_match(line, expected):
     assert actual == expected
 
 
+@mark.parametrize(("line", "expected"), [
+    (
+        (
+            "        inet 10.0.2.155"
+            "  netmask 10.0.2.1"
+            "  broadcast 10.0.2.15"
+        ),
+        (
+            "        inet 10.230.230.1"
+            "  netmask 10.230.230.3"
+            "  broadcast 10.230.230.2"
+        ),
```

Comment:
Now, the `ip addr` command output occurs multiple times in the tests. I’d consider extracting that to a constant or a generator function.